hack/build-and-push.sh

```diff
@@ -38,9 +38,11 @@ VCS_URL=https://github.com/konflux-ci/build-definitions
 VCS_REF=$(git rev-parse HEAD)
 
 declare -r ARTIFACT_TYPE_TEXT_XSHELLSCRIPT="text/x-shellscript"
-declare -r ANNOTATION_TASK_MIGRATION="dev.konflux-ci.task.migration"
 declare -r TEST_TASKS=${TEST_TASKS:-""}
 
+declare -r ANNOTATION_TASK_MIGRATION="dev.konflux-ci.task.migration"
```

Comment:
Does `dev.konflux-ci.task.migration-exists` makes it more clear?

Reply:
If we don't do it now, later it will be too late, if some real bundles would have a migration

Reply:
There is still space before the final delivery. Can we make this change in separate PRs for this script and the migration tool?

Reply:
yes, sure.